src/main/java/org/comic_conmuseum/fan_forge/backend/models/Survey.java

```diff
@@ -2,13 +2,35 @@
 
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
 
 public class Survey {
+    // These should _never_ be referenced directly if at all possible. It
+    // must also never, ever be edited. If you edit it, I will slap you.
+    public static final String[] POPULATIONS = {
+            "male", "female", "kids", "teenagers", "adults"
+    };
+    
```

Comment:
If you make `POPULATIONS` an enum, then you can 
 - still do the .values() to get all of them
 - create getter methods for:
    - how it's stored in the DB ("pop_" + pop + " BOOLEAN NOT NULL")
    - field name in the DB ("pop_" + pop)
    - the map for parameters in SQL (":pop_" + pop)
 - You could also create static methods on the enum that do the .map(...).collect(Join thing) to keep the code organized better

Reply:
ooh I knew there was a better way, thanks!